webhook_to_fedora_messaging/main.py

```diff
@@ -35,13 +31,11 @@ def create_app():
 
     # Then load the variables up from the custom configuration file
     try:
-       confdata = get_config()
+        confdata = get_config()
+        main.config.from_mapping(confdata)
```

Comment:
Do you expect the next three lines to raise ConfigError? If not, let's get them out of the block.